docs/quickstart.rst

```diff
@@ -323,6 +323,54 @@ Much like data objects, collection objects have a method :meth:`~WrightTools.col
 
 Collections can be saved inside of wt5 files, so be aware that :meth:`~WrightTools.open` may return a collection or a data object based on the contents of your wt5 file.
 
+Manipulating Your Data -- A 1D Plot Example
+----------------------
+Now that you have learned how to manipulate data provided within WrightTools itself, it is time to investigate manipulation of personal, experimental data. Above, data was already put in WrightTools and was in the .wt5 file format, which permitted use of 'wt.open()'. Here, this will no longer be the case, and other avenues must be explored for performing analysis in WrightTools. Below we will investigate plotting an infrared spectrum from a typical laboratory instrument (Bruker Tensor 27).
```

Comment:
```suggestion
Manipulating Your Data -- A 1D Plot Example
-------------------------------------------

Now that you have learned how to manipulate data provided within WrightTools itself, it is time to investigate manipulation of personal, experimental data. Above, data was already put in WrightTools and was in the .wt5 file format, which permitted use of 'wt.open()'. Here, this will no longer be the case, and other avenues must be explored for performing analysis in WrightTools. Below we will investigate plotting an infrared spectrum from a typical laboratory instrument (Bruker Tensor 27).
```

Reply:
just some formatting, this file format can be a little persnickety (markdown is better in this regard, but this is the standard for python docs, though that is changing)